readme.md

```diff
@@ -240,6 +240,12 @@ To reference the nested reference, please use "." delimiter, e.g.:
 * E.g. **shapeType** is resolved as "**B**" and would compose the **$shapeB** outer reference.
 * This resolution could be n-nested.
 
+#### Data Type Casting
+`$payment.amount.(Number)`
+
+-    This forces the data contexts `payment.amount` to be cast into a `number` before being used as an operand in the evaluation.
+-     Support data type cas ``(Number)`` <=> ``(String)``
```

Comment:
Suggestions: Just a small updates for clarity

`$payment.amount.(Type)`

Cast the given data context into the desired data type before being used as an operand in the evaluation.
> Note: If the conversion is invalid, the an warning message is being logged.

Supported data type conversions: 
- `.(String)`: cast a given reference to **String**.
- `.(Number)`: cast a given reference to **Number**.

Reply:
7185af3